tools/hardhat-example/scripts/deployContract.js

```diff
@@ -18,18 +18,14 @@
  *
  */
 
-const hre = require('hardhat');
-const hethers = require('@hashgraph/hethers');
+const { ethers } = require('hardhat');
 
 module.exports = async () => {
-  const provider = new hre.ethers.providers.JsonRpcProvider(process.env.RELAY_ENDPOINT);
-  const wallet = new hre.ethers.Wallet(process.env.OPERATOR_PRIVATE_KEY, provider);
-  const Greeter = await hre.ethers.getContractFactory('Greeter', wallet);
-  const greeter = await Greeter.deploy('initial_msg');
+  let wallet = (await ethers.getSigners())[0];
+  const Greeter = await ethers.getContractFactory('Greeter', wallet);
+  const greeter = await Greeter.deploy('initial_msg', { gasLimit: 1000000 });
```

Comment:
Why the explicit gasLimit here and above?
I sit just to indicate such calls don't cost more or is it required now for some reason?
If this needs to be explicit we should be clear why with a comment or some kind of an issue etc

Reply:
Actually, it's not needed. I've tested and probably forgot to remove it. Will do it now. Also ethers will automatically determine the gas, no need for explicitly writing the gasLimit.